Userland/Utilities/strings.cpp

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/CharacterTypes.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/Stream.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <unistd.h>
+
+enum class StringOffsetFormat {
+    None = 0,
+    Decimal,
+    Octal,
+    Hexadecimal
+};
+
+size_t minimum_string_length = 4;
+bool show_paths = false;
+auto string_offset_format = StringOffsetFormat::None;
```

Comment:
global state should be avoided, I don't see a reason why we can't pass them as arguments from the main function. Even if we couldn't:

https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#names

> Global variables should be prefixed by "g_".

Reply:
Eh, that's on me then. But I'd at least suggest wrapping those in a struct and passing that around as a reference, similar to what other utilities have done.

Reply:
> global state should be avoided

I already stated this, and was told that I was wrong. So I changed it. Now I'm being told to change it back. :shrug: